### ISSUE TITLE
Import membership lists from zoho

### ISSUE THREAD

**Issue by cfggns** [alternativelibraries/import-scripts#2]
For QPIRG-C, QPIRG-M and possibly the UGE. The default membership expiry should be made indefinite before doing the imports.

**Comment by cfggns** [comment]
I exported the QPIRG-C membership lists from Zoho but haven't done anything with them yet. From what I remember before with the QPIRG-M membership list, the PMB membership import feature had a nice interactive process that allowed assigning fields from the imported lists. I will need to figure out how to get an up-to-date version of the QPIRG-M membership list as well.

**Comment by cfggns** [comment]
I'm working with Nina on the membership import. QPIRG-C and M are both using Zoho, and have very similar fields. We decided to remove the preferred means of contact, backup contact (name, and email/phone), and affiliation to qpirg. We will keep "Time Added" even though it is only there now for mcgill, but rename it "Date Added" (assuming that this isn't tracked automatically by PMB?).

We also decided that the Membership numbers will start with C or M to designate the original library of the member. These will be added manually following the import. Therefore, QPIRG-M needs to finish with adding this code before the QPIRG-C import is initiated.

**Comment by cfggns** [comment]
We need to make sure that we're able to sort out members by library! The basic way will be doing it with the letter codes at the beginning of each member ID, but if there are other ways of doing this that are built into PMB that would be good!

**Comment by cfggns** [comment]
Another way to look at it is can we see which library a book is checked out from.

**Comment by mach-o** [comment]
Our preference was to have a separate library field in both the item and borrower tables and avoid any special prefixes

**Comment by cfggns** [comment]
I worked on this week and discussed it with Nina. Most of the problems with membership management between multiple libraries have been solved by having the members of each library in a "members group" with the contact for each group being the librarian for that library. Some more testing of how this works in practice is still required, but it seems so far to work pretty much the way we wanted in the last meeting where we talked about this. Nina and I agreed that this mean that individuals will have multiple memberships in PMB if they are borrowing from multiple libraries. 

Also, the members of QPIRG-M have been imported from Zoho. Now QPIRG-C remains. I will create a separate issue for figuring out membership imports from ResourceMate systems (the UGE for now, maybe CGA later).
